ironfist/src/cpp/gui/dialog.cpp

```diff
@@ -22,4 +23,14 @@ void DisplayError(const char* msg, const char* title) {
 	MessageBox(NULL, (LPCWSTR)wideCharMsg, (LPCWSTR)wideCharTitle, MB_OK);
 	FREE(wideCharMsg);
 	FREE(wideCharTitle);
-}
\ No newline at end of file
+}
+
+void ErrorSavingMapVariable(const char* mapVariableId) {

```

Comment:
We're adopting the std:string changes! Applies to all future code being merged in, this included :)